Composer/packages/lib/code-editor/src/lu/__tests__/DefineEntityButton.test.tsx

```diff
@@ -67,11 +64,6 @@ describe('<DefineEntityButton />', () => {
       jest.runAllTimers();
     });
 
-    await act(async () => {
-      fireEvent.click(screen.getByText('this page'));
-    });
-
-    expect(window.open).toBeCalled();
-    window.open = origOpen;
+    expect(screen.getByText('this page')).toBeInTheDocument();
```

Comment:
Fluent doesn't call `window.open` after it got updated. Manually verified the link does work:
![image](https://user-images.githubusercontent.com/2841858/157134704-b40f576b-239e-46d9-b009-991926e43dfc.png)